### PR TITLE
Add magic data from 2006

### PR DESCRIPTION
**Pull request by pdeiml**
Added two magic 2006 paper.

For myself:
Adapt the table in #157

### CODE REVIEW

File: input/data/2006/2006ApJ...639..761A/tev-000138.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+source_id: 138
+reference_id: 2006ApJ...639..761A
+telescope: magic
+data_entry_status: incomplete
+
+data:
+  livetime: 6h # The paper says '~6hours'
+  significance: 8.2
+  excess: 246
```
Comment (pdeiml) [inline input/data/2006/2006ApJ...639..761A/tev-000138.yaml]: The paper gives an error on excess but the schema does not allow errors on excess -> Add excess errors to the schema?

File: input/data/2006/2006ApJ...639..761A/tev-000138.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (pdeiml) [inline input/data/2006/2006ApJ...639..761A/tev-000138.yaml]: I think, no morph-type and no position is given in the paper, is it?

File: input/data/2006/2006ApJ...642L.119A/tev-000055.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+reference_id: 2006ApJ...642L.119A
+telescope: magic
+data_entry_status: incomplete
+
+data:
+  livetime: 8.2h
+  significance: 6.4
+  excess: 560
+
+pos:
```
Comment (pdeiml) [inline input/data/2006/2006ApJ...642L.119A/tev-000055.yaml]: Can you double check my entries for position and morph. Not completely clear to me. (page 3, left top).

File: input/data/2006/2006ApJ...642L.119A/tev-000055.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+morph:
+  type: gauss
+  sigma: {val: 0.06}
+
+
+spec:
+  model:
+    type: pl
+    parameters:
```
Comment (pdeiml) [inline input/data/2006/2006ApJ...642L.119A/tev-000055.yaml]: I don't understand the short paragraph about the errors on the spectrum.

Comment (cdeil) [reply input/data/2006/2006ApJ...642L.119A/tev-000055.yaml]: spectral values look OK to me.

File: input/data/2006/2006ApJ...642L.119A/tev-000055.yaml (hunk 3 above)
Comment (cdeil) [reply input/data/2006/2006ApJ...642L.119A/tev-000055.yaml]: This is an AGN, a point source. It doesn't have a measured extension and the paper also doesn't give a position measurement. Fixed.

File: input/data/2006/2006ApJ...639..761A/tev-000138.yaml (hunk 2 above)
Comment (cdeil) [reply input/data/2006/2006ApJ...639..761A/tev-000138.yaml]: This is an AGN. Yes, no position or morhphology was measured.

**Comment by cdeil** [comment]
Fixes in c545dd3 . Merging now.

@pdeiml - The data entry for these papers is incomplete. E.g. there's an SED given in Table 1 of http://iopscience.iop.org/article/10.1086/504845/pdf and there's a lightcurve given in Table 2 of http://iopscience.iop.org/article/10.1086/499421/pdf .
Please note "incomplete" when adding the `info.yaml` for these papers.

**Pull request merged**